tools/contractdeployer/obscuro_deployer.go

```diff
@@ -126,5 +126,5 @@ func (o *obscuroDeployer) TransactionReceipt(hash gethcommon.Hash) (*types.Recei
 }
 
 func getURL(cfg *Config) string {
-	return fmt.Sprintf("%s:%d", cfg.NodeHost, cfg.NodePort)
+	return fmt.Sprintf("ws://%s:%d", cfg.NodeHost, cfg.NodePort)
```

Comment:
Why hardcoding to ws ?

Reply:
For simplicity - everything currently is set up to point to the host's WS port. Since there will only be one contract deployer execution, there's no cost to keeping with WS.

Reply:
Fair enough 👍